types/openlayers/index.d.ts

```diff
@@ -9,6 +9,7 @@
 //                 Yair Tawil <https://github.com/yairtawil>
 //                 Pierre Marchand <https://github.com/pierremarc>
 //                 Hauke Stieler <https://github.com/hauke96>
+//                 Jan Suleiman <https://github.com/jansule>
```

Comment:
As explained in the DefinitelyTyped contributions guide, add yourself to the author list will cause you to be notified whenever someone makes a pull request. Is it really what you want?

Reply:
Good point there.